bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/fileformat/FileFormatResource.java

```diff
@@ -216,16 +226,28 @@ public Response createFileFormatForItem(final @Context HttpHeaders httpHeaders,
     @GET
     @RolesAllowed({ Role.ADMIN })
     @Path("/things")
-    @Produces("text/vnd.openhab.dsl.thing")
+    @Produces({ "text/vnd.openhab.dsl.thing", "application/yaml" })
```

Comment:
Is there a specific reason to choose `application/yaml`? If not, I'd suggest `text/yaml` to match with `text/vnd.openhab.dsl.thing`

Reply:
I checked this morning and discovered that application/yaml was standardized in 2024. text/yaml is now deprecated.